### PR TITLE
chore(opapi): publish instructions

### PR DESCRIPTION
**Pull request by spg**

### CODE REVIEW

File: opapi/readme.md [hunk 1]
```diff
@@ -92,3 +92,8 @@ The Botpress Engineering team.
 ## Licensing
 
 This software is protected by the same license as the [main Botpress repository](https://github.com/botpress/botpress). You can find the license file [here](https://github.com/botpress/botpress/blob/master/LICENSE).
+
+## Publishing
```
Comment (michaelmass) [inline opapi/readme.md:96]: **🎯 suggestion:** ‏ You should add this in the README.md at the root of the repository. I don't think we want to add the publishing steps of a npm package inside the npm README.
image

Comment (spg) [reply opapi/readme.md]: I added instructions to `opapi/CONTRIBUTING.md` since instructions may vary depending on package

Comment (michaelmass) [reply opapi/readme.md]: That's good but we should add it to the npm ignore `opapi/.npmignore`.

Comment (spg) [reply opapi/readme.md]: @michaelmass done in 9e4dcc6

**Pull request merged**